record-and-playback/core/lib/recordandplayback.rb

```diff
@@ -240,8 +240,17 @@ def self.rap_scripts_path
   def self.read_props
     return @props if @props
 
+    filepathRecOverride = "/etc/bigbluebutton/recording.yml"
+    isThereOverride = File.file?(filepathRecOverride)
+    
     filepath = File.join(BigBlueButton.rap_scripts_path, 'bigbluebutton.yml')
-    @props = YAML::load(File.open(filepath))
+    props = YAML::load(File.open(filepath))
```

Comment:
I'd propose to keep assigning the `load()` result to `@props`. Comes at no extra cost and gets rid of the `else`.

Reply:
I agree, I will change it there, thank you.